packages/map-template/src/components/ContactActionButton/ContactActionButton.jsx

```diff
@@ -0,0 +1,52 @@
+import PropTypes from "prop-types";
+import './ContactActionButton.scss'
+
+ContactActionButton.propTypes = {
+    detailType: PropTypes.string,
+    active: PropTypes.bool,
+    displayText: PropTypes.string,
+    value: PropTypes.string,
+    icon: PropTypes.string
+};
+
+/**
+ * 
+ * @param {object} props
+ * @param {string} props.detailType - The type of the contact detail (e.g. email, phone, website).
+ * @param {boolean} props.active - Whether the button should be displayed.
+ * @param {string} props.displayText - The text to display on the button.
+ * @param {string} props.value - The value of the contact detail (e.g. email address, phone number, website URL).
+ * @param {string} props.icon - The icon to display on the button.
+ * @returns 
+ */
+function ContactActionButton({ detailType, active, displayText, value, icon }) {
+    if (!active) return null;  // Early exit to not render the button if it is not set as active in the additional details data
+    const textToDisplay = detailType.toLowerCase() === 'phone' ? value : displayText; // If the detail type is a phone number, display the phone number instead of the display text
+
+    const handleClick = (detailType, value) => {
+        switch (detailType.toLowerCase()) {
+            case 'email':
+                window.location.href = `mailto:${value}`; // Opens default email client
+                break;
+            case 'phone':
+                window.location.href = `tel:${value}`; // Opens phone dialer
+                break;
+            default:
+                window.open(value, '_blank'); // Opens any other link in new tab
+                break;
+        }
+    };
+
+    return (
+        <button
```

Comment:
For semantic reasons, I think we should use an anchor element instead of a button when opening new locations. 

"When you're navigating the user to a page or resource, use an anchor.": https://davidwalsh.name/html5-buttons